app/src/main/java/edu/uc/kovaciad/slicetracker/MainActivity.kt

```diff
@@ -38,6 +39,7 @@ class MainActivity : AppCompatActivity() {
         setContentView(R.layout.main_activity)
         mainFragment = MainFragment.newInstance()
         overviewFragment = OverviewFragment.newInstance()
+        val viewModel = ViewModelProviders.of(this).get(MainViewModel::class.java)
```

Comment:
Instantiating ViewModel in MainActivity as shown in class material video here: https://youtu.be/RAQ5Bmx6YSc